common/lib/capa/capa/tests/test_capa_problem.py

```diff
@@ -150,7 +149,7 @@ def test_multiple_descriptions(self):
             }
         )
 
-    def test_default_question_text(self):
+    def test_empty_question_text(self):
```

Comment:
Can you remove this test, we're already checking empty text in other tests, so we don't need this.